### PR TITLE
Increase safe-wallet-web resources

### PR DESCRIPTION
**Pull request by ausias-armesto**
An OOMKilled error will occur if resources are lower than 4Gi

## Summary by CodeRabbit

- **New Features**
	- Updated the Helm chart version for the `safe-wallet-web` application to `0.1.6`.
	- Introduced specific resource limits and requests for the deployment, enhancing performance and reliability.
  
- **Improvements**
	- Simplified the deployment configuration by removing conditional logic for resource definitions, ensuring consistent resource allocation.
	- Added an initContainer for resource management at startup, optimizing the deployment process.

### CODE REVIEW

File: charts/safe-wallet-web/values.yaml [hunk 1]
```diff
@@ -33,18 +33,15 @@ commonAnnotations: {}
...
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-    # limits:
-    #   cpu: 100m
-    #   memory: 128Mi
-    # requests:
-    #   cpu: 100m
-    #   memory: 128Mi
+resources:
```
Comment (tolbrino) [inline charts/safe-wallet-web/values.yaml:38]: Can we keep the request small but the limit high? The problem is the large memory block is only used on startup, but not after that.

**Review by tolbrino — CHANGES_REQUESTED** [review]

File: charts/safe-wallet-web/values.yaml (hunk 1 above)
Comment (ausias-armesto) [reply charts/safe-wallet-web/values.yaml:38]: We can, but that will provoke more probability that the OOM will occur, as Kubernetes will be able to schedule the pod to more servers. And I will need to manually reschedule pods between servers so this pod can enter into a server with enough room. This same scenario happens on Forum where it compiles at the entrypoint. The solution would be to talk with Safe team and ask them to fix it.

Comment (tolbrino) [reply charts/safe-wallet-web/values.yaml:38]: ok

**Review by tolbrino — APPROVED** [review]

File: charts/safe-wallet-web/values.yaml (hunk 1 above)
Comment (ausias-armesto) [reply charts/safe-wallet-web/values.yaml:38]: I've come up with a workaround that might work. I've attached a dummy initContainer which allocates the required resources for compilation, so we assure that the Kubernetes node have space for it. The initContainer finishes immediately and the the main container only requests the low resources during normal operation and not for compiling.

**Pull request merged**